core/controllers/collection_editor_test.py

```diff
@@ -250,10 +278,20 @@ def test_publish_unpublish_collection(self):
             '%s/%s' % (
                 feconf.COLLECTION_URL_PREFIX, self.COLLECTION_ID))
         csrf_token = self.get_csrf_token_from_response(response)
+        url = '/collection_editor_handler/publish/%s' % collection_id
+        
+        # Check PUT returns 500 when an exception is raised validating exps
```

Comment:
Ditto, reg 500 responses. 

Could you point us to the line of code where the server error happens? We can try to rectify that.